src/App.tsx

```diff
@@ -13,8 +13,14 @@ class App extends React.Component<{}, State> {
     clockName: 0,
   };
 
-  clockVisibilityHandler = (isVisible: boolean): void => {
-    this.setState({ isClockVisible: isVisible });
+  showClock = (): void => {
+    this.setState({ isClockVisible: true });
+  };
+
+  hideClock = () => {
+    this.setState(
+      { isClockVisible: false },
+    );
```

Comment:
Why did you create these methods? Your clockVisibilityHandler method was really nice